core/templates/dev/head/pages/about/About.js

```diff
@@ -19,6 +19,19 @@
 oppia.controller('About', [
   '$scope', 'UrlInterpolationService',
   function($scope, UrlInterpolationService) {
+    $scope.listOfNames = ' Alex Kauffmann,' + ' Allison Barros,' +
```

Comment:
I tried doing this initially, but angular's ```ng-repeat``` (used to iterate through the list) does not seem to work when added as part of inline HTML inside the text for the i18n keys. I'd really appreciate some help on this. 
Thanks!